src/main/java/com/targomo/client/api/request/config/multigraph/AggregationConfig.java

```diff
@@ -0,0 +1,242 @@
+package com.targomo.client.api.request.config.multigraph;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.targomo.client.Constants;
+import com.targomo.client.api.enums.MultiGraphAggregationType;
+import org.apache.commons.lang3.builder.EqualsBuilder;
+import org.apache.commons.lang3.builder.HashCodeBuilder;
+
+import java.util.LinkedHashMap;
+import java.util.Map;
+import java.util.Set;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class AggregationConfig {
+    private MultiGraphAggregationType type;
```

Comment:
and shouldn't we have at least some basic test to see if variables are correctly set, hashcode/equals works and so on?

Reply:
I mean it is a POJO. those would be the first tests for "hash", "equals", etc. in all of our projects I think

Reply:
I think we should put a higher standard to new code then to the old one